coursemapper-kg/concept-map/src/pipelines/modify_graph.py

```
@@ -63,7 +64,8 @@
         graph.nodes.append(material_node)
 
         # Add concept node
-        concept_node = Node(f'{material_id}_concept_{str(abs(hash(str(wikipedia_page_embedding))))}', concept_name, '', 'main_concept', (concept_material_weight + 1) / 2, f'https://en.wikipedia.org/wiki/{concept_name}', wikipedia_page.summary, False, wikipedia_page_embedding, [])
+        concept_node = Node(f'{material_id}_concept_{str(abs(hash(str(wikipedia_page_embedding))))}', concept_name, '', 'main_concept', (concept_material_weight + 1) / 2, f'https://en.wikipedia.org/wiki/{concept_name}', wikipedia_page.summary, False, isNew , isEditing, lastEdited, wikipedia_page_embedding, []   )
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)